### PR TITLE
 Adding alternative magic method to retrieve arguments

### PR DESCRIPTION
**Pull request by rodrigowebjump**
## Purpose of this pull request

This pull request (PR) explain the alternative *get* magic method to retrieve arguments in template files

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html
https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rodrigowebjump** [comment]
@rogyar Done :)

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -373,24 +374,66 @@ These argument examples can be taken in the template like in the following examp
...
 $someString = $block->getData('some_string');
+
 /** @var bool $isActive */
 $isActive = $block->getData('is_active');
+
 /** @var Vendor\CustomModule\ViewModel\Class|\Magento\Framework\View\Element\Block\ArgumentInterface $viewModel */
 $viewModel = $block->getData('viewModel');
+
 /** @var string|int|float $someNumber */
 $someNumber = $block->getData('some_number');
```
Comment (drpayyne) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: Just a thought. Wouldn't it better, if the magic method examples were adjacent to their getData method examples? Might be more clear on the similarity and not partially duplicating a huge code block.

Comment (rodrigowebjump) [reply guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: Hi @drpayyne 

I did the adjustments combining the examples.
Thanks

**Comment by dobooth** [comment]
running tests

**Pull request merged**